src/common/OfflineNotice.js

```diff
@@ -36,15 +37,18 @@ class OfflineNotice extends PureComponent<Props> {
 
   render() {
     const { isOnline } = this.props;
-
-    if (isOnline) {
-      return null;
-    }
-
     return (
-      <View style={styles.block}>
-        <Label style={styles.text} text="No Internet connection" />
-      </View>
+      <AnimatedComponent
+        property="height"
+        useNativeDriver={false}
```

Comment:
What's the reason for choosing `false` here? `true` gets the work of updating the animation for each frame out of our JS thread, which sounds like a good thing.

Reply:
`useNativeDriver` doesn't work with layout props:
> Not everything you can do with Animated is currently supported in Native Animated. The main limitation is that you can only animate non-layout properties, things like transform and opacity will work but flexbox and position properties won't.

I tried using `transform`s `scaleY`, but it doesn't integrate as neatly as animating `height`:
![untitled](https://user-images.githubusercontent.com/7950151/42933758-dfa5bc6c-8b45-11e8-95f6-567b81f7dc6d.gif)

I'll add this explanation to the commit message.

> true gets the work of updating the animation for each frame out of our JS thread

After reading https://facebook.github.io/react-native/blog/2017/02/14/using-native-driver-for-animated, I the impression that that the main problem prior to the native driver was that an animation would drop frames if there was other heavy work done on the JS thread, and that every frame would need to be passed through the bridge. The post doesn't seem to associate heavy work with the animations themselves.

Reply:
OK, sounds good. Thanks for the explanation!